From d27e2374f2ac7b2dc41147e685e3f0d09c4f4b7e Mon Sep 17 00:00:00 2001
Message-ID: <d27e2374f2ac7b2dc41147e685e3f0d09c4f4b7e.1740173176.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1740173176.git.sharlatanus@gmail.com>
References: <cover.1740173176.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 20 Feb 2025 20:47:49 +0000
Subject: [PATCH 30/51] gnu: python-sep-pjw: Update to 1.3.8.

* gnu/packages/astronomy.scm (python-sep-pjw): Update to 1.3.8.
[propagated-inputs]: Add python-sep.

Change-Id: I3a98f356224fbd7bde28dd4a1bee61557c9d1cfd
---
 gnu/packages/astronomy.scm | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 8664d073a7f..48df06b008e 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -6155,13 +6155,13 @@ (define-public python-sep
 (define-public python-sep-pjw
   (package
     (name "python-sep-pjw")
-    (version "1.3.7")
+    (version "1.3.8")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "sep_pjw" version))
        (sha256
-        (base32 "1z8m2l3ccxj1izdazv7a0hg6zd1495bvby8w114s5ji7fd91vdv7"))))
+        (base32 "0lhxfq1acc6qc8nszfdrpwq6dizaypz3b6frknfv5qm59mb488r0"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -6173,7 +6173,8 @@ (define-public python-sep-pjw
            python-setuptools-scm-next
            python-wheel))
     (propagated-inputs
-     (list python-numpy))
+     (list python-numpy
+           python-sep))
     (home-page "https://github.com/PJ-Watson/sep-pjw")
     (synopsis "Alternative fork of SEP library")
     (description
-- 
2.47.1

